parquet/src/file/properties.rs

```diff
@@ -27,7 +27,7 @@
 //! };
 //!
 //! // Create properties with default configuration.
-//! let props = WriterProperties::builder().build();
+//! let props = WriterProperties::default();
```

Comment:
```suggestion
//! let props = WriterProperties::new();
```

See suggestion below